include/oneapi/tbb/task_group.h

```diff
@@ -779,13 +693,16 @@ class isolated_task_group : public task_group {
 
 #if __TBB_PREVIEW_TASK_GROUP_EXTENSIONS
     void run(d2::task_handle&& m_handle) {
-        if (m_handle == nullptr)
+        if (m_handle == nullptr) {
```

Comment:
Rename `m_handle` to `h` in this function to be consistent with `task_group::run()`?
Otherwise, looks strange that some member ("m_") variable belongs to public interface.

Reply:
renamed